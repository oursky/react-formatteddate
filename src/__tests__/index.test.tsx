import * as React from "react";
import * as renderer from "react-test-renderer";
import { FormattedDate } from "../index";

test("FormattedDate", () => {
  const value = new Date("2006-01-02T15:04:05+08:00");
  const format = "LLLL";
  const tree = renderer
    .create(<FormattedDate format={format} value={value} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
