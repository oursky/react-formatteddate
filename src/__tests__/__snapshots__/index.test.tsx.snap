// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormattedDate 1`] = `"Monday, January 2, 2006 3:04 PM"`;
